src/gilbert/site.py

```diff
@@ -79,10 +79,50 @@ def init(self):
         self.dest_dir.mkdir(parents=True, exist_ok=True)
 
     def load_plugins(self):
+        """
+        Load the plugins.
+
+        Note that the default provided plugins are loaded first so that any custom plugins
+        that are loaded later are able to override the defaults.
+        """
         from . import plugins
 
         found = OrderedDict()
 
+        # Attempt to lead the deafult plugins first
+        DEFAULT_PLUGIN_FILES = [
```

Comment:
I'd rather not hard-code this list in the file, as it requires we then maintain it later if/when plugins are added/removed.

Reply:
I agree that it would be best to not have this hard-coded here.